src/common/ecmdDefines.H

```diff
@@ -98,7 +98,8 @@
 //#define ECMD_REMOVE_MEMORY_FUNCTIONS
 //#define CIP_REMOVE_MEMORY_FUNCTIONS
 //#define ECMD_REMOVE_INIT_FUNCTIONS
-#endif // EDBG_BUILD_HOST
+#define EDBG_ISTEP_CTRL_FUNCTIONS
```

Comment:
I don't think this define is deployed quite right in the build.  I need to think about it more.

Reply:
OK

Reply:
Ok

Reply:
Let me know how to proceed if there is any better way than this